configure.ml

```diff
@@ -623,23 +623,23 @@ let check_camlp5_version camlp5o =
     printf "You have Camlp5 %s. Good!\n" version; version
   | _ -> die "Error: unsupported Camlp5 (version < 6.06 or unrecognized).\n"
 
-let config_camlpX () =
+let config_camlp5 () =
     let camlp5mod = "gramlib" in
     let camlp5libdir, camlp5o = check_camlp5 (camlp5mod^".cma") in
     let camlp5_version = check_camlp5_version camlp5o in
-    "camlp5", "Camlp5", camlp5o, Filename.dirname camlp5o, camlp5libdir, camlp5mod, camlp5_version
+    "Camlp5", camlp5o, Filename.dirname camlp5o, camlp5libdir, camlp5mod, camlp5_version
 
-let camlpX, capitalized_camlpX, camlpXo,
-    camlpXbindir, fullcamlpXlibdir,
-    camlpXmod, camlpX_version = config_camlpX ()
+let capitalized_camlp5, camlp5o,
```

Comment:
I would also (consistently) remove the `capitalized_camlp5`. I don't know what the story of this variable is but it looks like it serves no special purpose today and its constant value could be in-lined like you did for `camlp5`.